src/ui/services/customers/add-new-customer.ui-service.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, Page } from "@playwright/test";
+import { apiConfig } from "config/api-config";
+import { generateCustomerData } from "data/salesPortal/customers/generateCustomer.data";
+import { STATUS_CODES } from "data/salesPortal/statusCodes";
+import { ICustomer, ICustomerResponse } from "types/salesPortal/customer.types";
+import { AddNewCustomerPage } from "ui/pages/customers/add-new-customer.page";
+import { CustomersPage } from "ui/pages/customers/customers.page";
+import _ from "lodash";
+
+export class AddNewCustomerUiService {
+  private addNewCustomerPage: AddNewCustomerPage;
+  private customersPage: CustomersPage;
+  constructor(private page: Page) {
+    this.addNewCustomerPage = new AddNewCustomerPage(page);
+    this.customersPage = new CustomersPage(page);
+  }
```

Comment:
Как вариант сделать класс `PageHolder` которую можно наследовать любому классу, который использует `Page`

```suggestion
export class PageHolder {
  constructor(protected page: Page) {}
}

export class AddNewCustomerUiService extends PageHolder {
  private addNewCustomerPage: AddNewCustomerPage = new AddNewCustomerPage(this.page);
  private customersPage: CustomersPage = new CustomersPage(this.page);
```

Reply:
done
ток хз в ту ли папку засунул
или лучше создать что-то типо holders

Reply:
Это считается общей пейджой (условно), можно просто рядом к базовой